tests/lora/test_lora_manager.py

```diff
@@ -120,7 +129,7 @@ def test_lora_model_manager(dist_init, dummy_model):
         2,
         2,
         LoRAConfig(max_lora_rank=8, max_cpu_loras=3, max_loras=2),
-        lora_target_modules=["dense1", "dense2", "lm_head"])
+        support_lora_modules=["dense1", "dense2", "lm_head"])
```

Comment:
Can you rename this to `supported_lora_modules`? The idea here is this is a list of modules that can be LoRAified (i.e. the list of modules for which the model supports applying LoRA adapters).